packages/utilities/psammead-locales/moment/helpers/jalaali.js

```diff
@@ -0,0 +1,30 @@
+const moment = require('moment-timezone/moment-timezone');
+const jalali = require('jalaali-js');
+
+function getJalaliString(gregorianMoment, jalaliMonths) {
```

Comment:
```suggestion
function getJalaaliDatetime(gregorianMoment, jalaliMonths) {
```
Can you please export this function directly. 
It will be good to add a few unit tests for this. For both fa and ps locales using this Jalaali helper. 
E.g. With an input date of 31-01-2019, should get for fa locale:

<span dir="rtl">۱۱ بهمن ۱۳۹۷</span>
And should get, for ps locale:
<span dir="rtl">۱۱ سلواغه ۱۳۹۷</span>

Reply:
Do you think the existing tests such as https://github.com/bbc/psammead/pull/1844/files#diff-96d07ccc729cfbdc689e14884f8452c9R47-R60 provide sufficient coverage for the helper itself?

I would agree the tests above could be rewritten to be more generic as the helper is actually agnostic to the locale and format, but specific coverage for Persian and Pashto is provided by `fa.test.js` and `pa.test.js`.